raiden/settings.py

```diff
@@ -55,7 +55,7 @@
 
 MIN_REI_THRESHOLD = 100
 
-MONITORING_REWARD = TokenAmount(1)
+MONITORING_REWARD = TokenAmount(5_000_000_000_000_000_000)  # about 1$
```

Comment:
This depends on the price of RDN no?

Reply:
Sure, the question is if we need an oracle, or if it's enough when we update this with every release.

Reply:
We would definitely need an Oracle. Price fluctuations happen so often that even daily releases would not help here.

Reply:
I would not go for an oracle before we publish the RC. That sounds like a lot of legal and testing effort. However, the monitoring service is not economical viable anyway at the moment and we will not change that by exactly providing 1$ for every successful defeat or not. So we should go with the easiest solution and change it iteratively after the release.